CODEOWNERS

```diff
@@ -653,6 +653,7 @@
 /modules/hal_gigadevice/                  @nandojve
 /modules/hal_nordic/nrf_802154/           @jciupis
 /modules/trusted-firmware-m/              @microbuilder
+/sysbuild/                                @tejlmand
```

Comment:
I guess you should switch to MAINTAINERS now

Reply:
I guess we need one each place in order for github to also work properly, right ?

as there is no entry for `/share` itself.

Reply:
MAINTAINERS already has:
https://github.com/zephyrproject-rtos/zephyr/blob/bc0d0a390613ca16720441201ba6190c08a974c1/MAINTAINERS.yml#L259-L275

and as sysbuild is now under `/share` it's already covered.